noir-projects/noir-protocol-circuits/crates/types/src/type_serialization.nr

```diff
@@ -82,6 +82,7 @@ impl Serialize<U128_SERIALIZED_LEN> for U128 {
         //   first.
         // For this reason if we didn't use the ordering of U128 limbs here and in encoder.ts we would get an arguments
         // hash mismatch.
+        // The below warning is due to visibility in noir stdlib.
         [self.lo, self.hi]
```

Comment:
This is the only warning in `noir-protocol-contracts` :'(

Reply:
This is fine. We'll very likely be replacing `U128` with a native `u128` in future so we can leave this until then.